nixos/modules/services/mail/stalwart-mail.nix

```
@@ -7,11 +7,28 @@ let
   configFormat = pkgs.formats.toml { };
   configFile = configFormat.generate "stalwart-mail.toml" cfg.settings;
   dataDir = "/var/lib/stalwart-mail";
+  stalwartGreaterOrEqualThan = versionAtLeast cfg.package.version;
```

```suggestion
  stalwartAtLeast = versionAtLeast cfg.package.version;
```
lets name this like the function